tests/Microsoft.Identity.Client.Performance/GetAllAccessTokensTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.Identity.Test.Unit;
+
+namespace Microsoft.Identity.Test.Performance
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Usage", "UseConfigureAwait:Use ConfigureAwait", Justification = "Test project, inapplicable.")]
```

Comment:
We tend to apply this rule in tests as well. It's good practice. You can always use `.ConfigureAwait(true)` which is the same as not using it.

Reply:
Done.